kitchen.docker.yml

```diff
@@ -0,0 +1,33 @@
+---
+driver:
+  name: dokken
+  pull_platform_image: false
+  chef_version: 17.2.29
+  env:
+    - KERNEL_RELEASE=3.10.0-1127.8.2.el7.x86_64
```

Comment:
I would explain that this is the format we validate `KERNEL_RELEASE` against, while on docker it is different, so that line of code breaks on docker.

Reply:
Done